samples/msgext-northwind-inventory-python/lab/Exercise 02 - Run sample app.md

```diff
@@ -0,0 +1,260 @@
+
+The resulting **.env** file should contain this line:
+
+~~~text
+STORAGE_ACCOUNT_CONNECTION_STRING=UseDevelopmentStorage=true
+~~~
+
+![Copy .env.sample to .env](./images/02-01-Setup-Project-01.png)
+
+## Step 2 - Run the application locally
+
+Press **CTRL+Shift+P** to open the command box and enter **Python: Create Environment** to create and activate your desired virtual environment. 
+
+![Create Environment](./images/02-02-Create-Environment-01.png)
+
+Select **Venv** from the search results.
+
+![Select Venv](./images/02-02-Select-Venv-01.png)
+
+Select the version of Python that is installed on the machine.
+
+![Select Python version](./images/02-02-Select-Python-version-01.png)
+
+Remember to select `requirements.txt` as dependencies to install when creating the virtual environment.
+
+![Select requirements](./images/02-02-Select-requirements.png)
+
+Click F5 to start debugging, or click the start button 1️⃣. You will have an opportunity to select a debugging profile; select Debug in Teams (Edge) 2️⃣ or choose another profile.
+
+![Run application locally](./images/02-02-Run-Project-01.png)
+
+If you see this screen, you need to fix your **.env** file; this is explained in the previous step.
+
+![Error is displayed because of a missing environment variable](./images/02-01-Setup-Project-06.png)
+
+The first time your app runs, you may be prompted to allow NodeJS to go through your firewall; this is necessary to allow the application to communicate.
+
+It may take a while the first time as it's loading all the npm packages. Eventually, a browser window will open and invite you to log in.
+
+![Browser window opens with a login form](./images/02-02-Run-Project-03.png)
+
+Once you're in, Microsoft Teams should open up and display a dialog offering to install your application.
+Take note of the information displayed; which is from the [app manifest](../appPackage/manifest.json).
+
+Click "Add" to add Northwind Inventory as a personal application.
+
+![App installation screen with large Add button](./images/02-02-Run-Project-04.png)
+
+You should be directed to a chat within the application, however you could use the app in any chat.
+
+## Step 3 - Test in Microsoft Teams
+
+In any Teams chat - including the Northwind Inventory chat - begin typing a message 1️⃣ that refers to a product. Then, to insert an adaptive card for the product, click the + 2️⃣ . In the fly-up panel, select the Northwind Inventory application you just installed 3️⃣ .
+
+![Clicking the "+" to open the message extension panel](./images/02-03-TestME-Teams-take2-01.png)
+
+You will see a search dialog with 3 tabs 1️⃣. The "Product Inventory" tab allows searching for products by name; for now, just type a product name, or the beginning of a product name such as "chai" into the search box 2️⃣ . If you pause as you type the first letters, you'll be given more choices of products that begin with the same characters.
```

Comment:
I get an error at this point, most likely due to the syntax error.

Reply:
Fixed